knox-token-management-ui/token-management/app/token.management.component.ts

```diff
@@ -30,7 +30,9 @@ import {MatSort} from '@angular/material/sort';
 
 export class TokenManagementComponent implements OnInit {
 
-    tokenGenerationPageURL = window.location.pathname.replace(new RegExp('token-management/.*'), 'token-generation/index.html');
+    pathParts = window.location.pathname.split('/');
```

Comment:
These lines smell like a utility method to me.

Reply:
I couldn't agree more, but I did not find out (yet) how to do that properly in TypeScript between different Maven modules. Let's leave it as-is for now and refactor this out when I'll have time to investigate.